samples/secret/secret.yaml

```diff
@@ -51,6 +51,20 @@ arrays:
     # Default Value: None
     nasName: "nas-server"
 
+    # nfsAcls: enables setting permissions on NFS mount directory
+    # This value will be used if a storage class does not have the nfsAcls parameter specified
```

Comment:
Change `nfsAcls` to `NFS ACL` in both sample files.

Reply:
done